src/components/LeftNavBar.js

```diff
@@ -1,34 +1,38 @@
 import { BsFillPersonFill, TbTextDirectionRtl, TbLayoutDashboard,
   IoNotificationsSharp, MdBackupTable, GiBulletBill, SiVirtualbox, RxDashboard, ImEnter, AiOutlineForm,
 } from '../utils/icons';
+import "./LeftNavBar.css"
 
 export default function LeftNavBar() {
- let navItemCSS = 'flex items-center py-3 hover:bg-neutral-900 rounded-lg active:bg-pink-600'
-
+ let navItemCSS = 'flex items-center py-3 hover:bg-neutral-900 rounded-lg active:bg-pink-600 mx-4'
+ let iconCSS = "p-1 text-3xl"
+ 
   return (
-    <div className="w-screen grid grid-cols-2 border-[2px] border-white ">
-      <div className="w-[180px] h-[calc(100%-4px)] fixed top-[2px] left-5 border-[2px] border-black bg-gradient-to-b from-neutral-800 to-neutral-950 rounded-lg">
-        <div className="mx-4 text-white text-sm">
-          <div className='flex items-center py-3 border-b-2 mb-2'><TbLayoutDashboard className="p-1" />Material Dashboard </div>
-          <div className="block h-[calc(50vh)] overflow-auto overscroll-y-auto">
+    <div className="w-screen grid grid-cols-2 border-[2px] border-white">
```

Comment:
Eventually this left navbar will need to placed beside the main content, but if you do this with your current `w-screen` i reckon it'll mess things up for the overall layout.

```
<LeftNavBar />
<MainContent />
```

^ You wouldn't want to move the main content over by the whole screen width. I think this one will have to change perhaps.

Reply:
I think I can better address this when the two components are merged together but I will flag it in VS Code 👍  Thanks for the heads up